GitUI/CommandsDialogs/FormBrowse.InitRevisionGrid.cs

```diff
@@ -39,11 +39,10 @@ private void InitRevisionGrid(ObjectId? selectedId, ObjectId? firstId, bool isBl
 
             RevisionGrid.RevisionsLoading += (sender, e) =>
             {
-                // The FileTree tab should be shown at first start, in "filehistory" mode
-                if (isBlame)
+                // Open diff in "filehistory" mode
+                if (isFileHistory)
                 {
```

Comment:
```
            bool firstTimeInFileHistoryMode = isFileHistoryMode;
            RevisionGrid.RevisionsLoading += (sender, e) =>
            {
                // Open diff in "filehistory" mode
                if (firstTimeInFileHistoryMode)
                {
                    firstTimeInFileHistoryMode = false;
```

Reply:
Is this really necessary?

Reply:
Yes, because changing tabs on `F5` or other refreshes is a nuisance.